unittests/util/TestObject.h

```diff
@@ -284,8 +296,28 @@ CreateAMMObject(
     std::string_view assetCurrency,
     std::string_view assetIssuer,
     std::string_view asset2Currency,
-    std::string_view asset2Issuer
+    std::string_view asset2Issuer,
+    std::string_view lpTokenBalanceIssueCurrency = "03930D02208264E2E40EC1B0C09E4DB96EE197B1",
+    uint32_t lpTokenBalanceIssueAmount = 100u,
+    uint16_t tradingFee = 5u,
+    uint64_t ownerNode = 0u
+);
+
+void
+AMMAddVoteSlot(ripple::STObject& amm, ripple::AccountID const& accountId, uint16_t tradingFee, uint32_t voteWeight);
+
+void
+AMMSetAuctionSlot(
+    ripple::STObject& amm,
+    ripple::AccountID const& accountId,
+    ripple::STAmount price,
+    uint16_t discountedFee,
+    uint32_t expiration,
+    std::vector<ripple::AccountID> const& authAccounts = {}
 );
```

Comment:
Probably at least for now we don't need these methods anywhere except AMMInfo tests. So maybe it would be better for now to place them into `AMMInfoTests.cpp`? We can always move them out when we need them somewhere else.

Reply:
Hmm. Not sure tbh. as TestObject files were exactly for this - to support all test stuff. 
It's like a collection of utils for all cases in our UT life 😃